src/DETHRACE/common/graphics.c

```diff
@@ -1188,10 +1188,11 @@ int ConditionallyFillWithSky(br_pixelmap* pPixelmap) {
     int bgnd_col;
     LOG_TRACE("(%p)", pPixelmap);
 
-    if (gProgram_state.current_depth_effect.sky_texture
-        && (!gLast_camera_special_volume || gLast_camera_special_volume->sky_col < 0)) {
-        return 0;
-    }
+    // TODO: Uncomment when ExternalSky is fully implemented
```

Comment:
Also add a `WARN_ONCE`. That way it remains visible.

Reply:
good call, done